packages/node_modules/@node-red/nodes/core/common/60-link.html

```diff
@@ -59,7 +59,7 @@
                     search.searchBox("count","");
                 } else {
                     var count = treeList.treeList("filter", function(item) {
-                        return item.label.indexOf(val) > -1
+                        return item.label.indexOf(val) > -1 || item.node.type.indexOf(val) > -1
```

Comment:
Strictly speaking, the link node list is already all of the same type... but no harm being consistent